content/en/logs/log_configuration/indexes.md

```diff
@@ -75,9 +75,15 @@ By default, logs indexes have no exclusion filter: that is to say all logs match
 
 But because your logs are not all and equally valuable, exclusion filters control which logs flowing in your index should be removed. Excluded logs are discarded from indexes, but still flow through the [Livetail][8] and can be used to [generate metrics][9] and [archived][10].
 
+To add an exclusion filter:
+
+1. Navigate to [Log Indexes][11].
+2. Expand the pipeline ƒor which you want to add an exclusion filter. 
```

Comment:
```suggestion
2. Expand the pipeline for which you want to add an exclusion filter. 
```
Looks like a special character snuck in there somehow.

Reply:
Lol strange, good catch!